applications/application-tsm-mac-verification.md

```diff
@@ -0,0 +1,201 @@
+
+## 1.7. Scope 
+
+Mining Association of Canada (MAC) is a participant in an open ecosystem and the focus of this framework is to describe the process MAC uses for digital exchanges on Indy networks.
+
+## 1.8. Objectives
+
+_This section states the high-level outcomes desired by the trust community through its adoption of the GF._
+
+1. SHOULD specify tangible, achievable results (e.g. SMART criteria and Fit-for-Purpose criteria).
+2. MUST only contain outcomes over which the GF has the authority and mechanisms to achieve within its
+scope.
+3. MUST be consistent with the principles of the GF (below).
+
+
+## 1.9. Principles 
+
+The Towards Sustainable Mining (TSM) program created by the MAC has a set of [Guiding Principles](https://mining.ca/download/36658/), which all participating members must commit to. These [Guiding Principles](https://mining.ca/download/36658/), address the expectation of its members towards adopting social, economic and environmental practices that align with the priorities and values of their communities of interest. 
+
+## 1.10. General Requirements
+TBD
+
+## 1.11. Revisions
+
+_TBD - How and when Mining Association of Canada (MAC) would change governance framework_
+
+## 1.12. Extensions
+
+There are no extensions to this Governance Framework.
+
+## 1.13. Schedule of Controlled Documents
+
+TBD
+
+# 2. Controlled Documents
+
+## 2.1. Glossary
+N/A
+
+## 2.2. Risk Assessment
+
+TBD
+
+## 2.3. Trust Assurance and Certification
+
+TBD
+
+## 2.4. Governance Requirements
+
+TBD
+
+## 2.5. Business Requirements
+Participation in the TSM initiative is mandatory for all MAC members for their Canadian operations, members MUST adhere to the [TSM reporting requirements](https://mining.ca/towards-sustainable-mining/protocols-frameworks/tsm-protocol-reporting-schedule/):
+
+* Self-assessed TSM performance indicators MUST be reported to MAC annually. Amendments to this requirement are offered to high TSM performers and described in the [TSM Primer]( https://mining.ca/wp-content/uploads/dlm_uploads/2021/04/TSM-Primer-English.pdf) under the section labelled _Optional Reporting and Verification Process_
+
+* Every three years, a Verification Service Provider (VSP) critically reviews a company’s self-assessments to determine if there is adequate evidence to support the performance ratings reported. A list of VSPs is provided [here](https://mining.ca/wp-content/uploads/dlm_uploads/2022/08/2022-August-TSM-Verifier-List.pdf).
+
+* In the year of external verification, a company’s CEO, or equivalent, MUST to submit a letter to MAC that confirms the external verification has been conducted in accordance with the Terms of Reference for VSPs.
```

Comment:
I think this needs to be updated to describe the new proposed process

Reply:
Here is my first attempt edit: 
![image](https://user-images.githubusercontent.com/122049466/222587731-f5450cbd-7970-4101-acd3-fd369203373e.png)